src/internal/connector/onedrive/collection_test.go

```diff
@@ -600,6 +605,19 @@ func (suite *GetDriveItemUnitTestSuite) TestGetDriveItem_error() {
 	}
 }
 
+var _ urlCacher = &mockURLCache{}
```

Comment:
Is it possible to move this to `onedrive/mocks`?  We want to start thinking about portability for mocks, whenever possible.